public/app/assetlibrary/item/item.html

```diff
@@ -26,7 +30,7 @@
     <div data-ng-if="asset.type !== 'whiteboard'">
 
       <!-- COMPLETED PREVIEWS -->
-      <div class="text-center assetlibrary-item-preview" id="assetlibrary-item-preview" data-ng-show="asset.preview_status === 'done'">
+      <div class="text-center assetlibrary-item-preview" id="assetlibrary-item-preview" data-ng-if="asset.preview_status === 'done'">
         <!-- Documents -->
```

Comment:
**Note:** I changed `ng-show` to `ng-if` to get consistent with all other preview_status related logic in the template. These sections do not show/hide per background processes so why use `ng-show`?